lib/stdlib/src/erl_lint.erl

```diff
@@ -3984,13 +3986,17 @@ extract_sequence(4, [$t, $w | Fmt], Need) ->
 extract_sequence(4, [$t, $W | Fmt], Need) ->
     extract_sequence(5, [$W|Fmt], Need);
 extract_sequence(4, [$t, C | _Fmt], _Need) ->
-    {error,"invalid control ~t" ++ [C]};
+    {error,"invalid modifier ~t" ++ [C]};
 extract_sequence(4, [$l, $p | Fmt], Need) ->
     extract_sequence(5, [$p|Fmt], Need);
+extract_sequence(4, [$l, $t, $p | Fmt], Need) ->
+    extract_sequence(5, [$p|Fmt], Need);
 extract_sequence(4, [$l, $P | Fmt], Need) ->
     extract_sequence(5, [$P|Fmt], Need);
+extract_sequence(4, [$l, $t, $P | Fmt], Need) ->
+    extract_sequence(5, [$P|Fmt], Need);
 extract_sequence(4, [$l, C | _Fmt], _Need) ->
-    {error,"invalid control ~l" ++ [C]};
+    {error,"invalid modifier ~l" ++ [C]};
```

Comment:
C is no longer always a control. ltc in a format string, for example.

Reply:
Right, I missed that the error message appends the rest of the string. Restoring it to the old way (but if this code gets rewritten, it would be nice to point out more exactly that the problem is that the modifier is not allowed with the main control character).